travis/compile.sh

```diff
@@ -25,3 +25,13 @@ fi
 meson . build ${MESON_OPTION}
 cd build
 ninja
+
+# Generate DMG
+if [[ "${TRAVIS_OS_NAME}" == "osx" ]]
+then
+  cd ..
+  mkdir zimwriterfs
+  cp -r build/src/* ./zimwriterfs
```

Comment:
Why copy all the src content ?
We only need the `zimwriterfs` binary no ?
But, we are compiling dynamically on osx, so we probably need to package the library dependencies also.
I'm not sure how appdmg does it (if it does it)